packages/mux-uploader/README.md

```diff
@@ -77,9 +74,9 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 | `url`                | `string`  | The authenticated URL that your file will be uploaded to. Check out the [direct uploads docs](https://docs.mux.com/guides/video/upload-files-directly#1-create-an-authenticated-mux-url) for how to create one. Required. | `undefined` |
 | `id`                 | `string`  | An ID that allows `mux-uploader-drop` to locate `mux-uploader`. Required.                                                                                                                                                 | N/A         |
 | `type`               | `"bar"`   | Specifies the visual type of progress bar. A radial type is in-progress.                                                                                                                                                  | "bar"       |
-| `upload-in-progress` | `boolean` | The thumbnail token for signing the `poster` URL.                                                                                                                                                                         | false       |
-| `upload-error`       | `boolean` | The storyboard token for signing the storyboard URL.                                                                                                                                                                      | false       |
-| `status`             | `boolean` | This is an arbitrary title for your video that will be passed in as metadata into Mux Data. Adding a title will give you useful context in your Mux Data dashboard. (optional, but encouraged)                            | false       |
+| `upload-in-progress` | `boolean` | Toggles visual status of progress bar while upload is n progress.                                                                                                                                                         | false       |
+| `upload-error`       | `boolean` | Toggles visual status of progress bar when upload encounters an error.                                                                                                                                                    | false       |
+| `status`             | `boolean` | Toggles text status of progress bar, which is a percentage by default.                                                                                                                                                    | false       |
```

Comment:
looks better 👍🏼 

Can you clarify how someone would use these attributes? As a developer what is the use case for when I would want to show or hide the progress bar or error?

Or are these attributes that get sent internally by the web component and I'm not supposed to be manipulating them from the outside?

I'm not quite following `status` which says is `boolean` but then says it is a percentage by default. What is the alternative to percentage?

Reply:
Will work on clarifying! Thanks for providing insight as a person totally new to the docs.